<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addProductButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeProductButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Selected Product</string>
    </property>
   </widget>
   <widget class="QListWidget" name="cartListWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="totalLabel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Total: $0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
